### PR TITLE
Add registry team to OWNERS of end-to-end tests

### PR DESCRIPTION
**Pull request by dmage**

### CODE REVIEW

File: test/end-to-end/OWNERS [hunk 1]
```diff
@@ -21,3 +21,6 @@ approvers:
   - knobunc
   - dcbw
   - danwinship
+  - legionus
+  - miminar
+  - dmage
```
Comment (kargakis) [inline test/end-to-end/OWNERS]: Add yourself as a reviewer too :)

Comment (dmage) [reply test/end-to-end/OWNERS]: Done

**Comment by dmage** [comment]
flake #16168
/retest

**Pull request merged**